Add task instance clarification (#3117)
astronomer/docs: astro/alerts.md

2. Click **Add Alert**. 

3. Enter your **Alert Name** and choose the alert type, **DAG Success**, **DAG Failure**, or **Task Duration**.

>   :::tip

Reviewer: ```suggestion
  :::info
```

>   You can only use Task Duration alerts with Task Instances, not Task Groups.

Reviewer: ```suggestion
  You can only use Task Duration alerts with individual tasks. Alerting on task group duration is not supported. 
```
- Task instance often means "task run" in OSS, which isn't quite what this is. We're talking more generally about the task itself, not the individual runs.

    - **DAG success**: Click **DAG** and choose the Deployment and the DAG that you want to send an alert about when it completes.
    

>     - **Task duration**: Click **Task** and choose the Deployment, DAG, and task instance name. Enter the **Duration** for how long a task should take to run before you send an alert to your communication channels.

Reviewer: ```suggestion
    - **Task duration**: Click **Task** and choose the Deployment, DAG, and task name. Enter the **Duration** for how long a task should take to run before you send an alert to your communication channels.
```
